pom.xml

```
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>edu.whimc</groupId>
 	<artifactId>WHIMC-Observations</artifactId>
-	<version>2.4.5</version>
+	<version>2.4.6</version>
```

```suggestion
	<version>2.5.0</version>
```
I'd say this is good to be a minor version bump